### PR TITLE
Make bf16 default for hpu

### PR DESCRIPTION
**Pull request by abhilash1910**
# What does this PR do?

Makes bf16 default dtype for models (if not set , gets set to None ) for all models, unless overridden with --dtype arg. 

cc @libinta @mandy-li @regisss 
@OlivierDehaene OR @Narsil

 -->

### CODE REVIEW

File: launcher/src/main.rs [hunk 1]
```diff
@@ -501,13 +501,12 @@ fn shard_manager(
 
     if let Some(dtype) = dtype {
         shard_args.push("--dtype".to_string());
-        shard_args.push(dtype.to_string())
+        shard_args.push(dtype.to_string());
```
Comment (schoi-habana) [inline launcher/src/main.rs:504]: does it fail for you without the semicolon?

Comment (abhilash1910) [reply launcher/src/main.rs:504]: No it does not fail as 504L is a terminal line . That change is optional though. Let me rephrase the header.

**Review by tthakkal — APPROVED** [review]

**Review by schoi-habana — APPROVED** [review]

**Review by pk1d3v — APPROVED** [review]

**Review by mgonchar — APPROVED** [review]
LGTM

**Review by leopck — APPROVED** [review]
LGTM

**Review by regisss — APPROVED** [review]
LGTM
